doc/api/path.md

```diff
@@ -135,6 +135,18 @@ process.env.PATH.split(path.delimiter);
 // Returns ['C:\\Windows\\system32', 'C:\\Windows', 'C:\\Program Files\\node\\']
 ```
 
+## `path.devnull`
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+Provides the platform-specific file path of the null device:
+
+* `\\.\nul` on Windows
```

Comment:
https://docs.python.org/3/library/os.html#os.devnull says:

> '/dev/null' for POSIX, 'nul' for Windows.

So, why isn't this just `'nul'` for Windows?

Reply:
Because if you do something like

```js
fs.writeFileSync(path.devnull, 'foo');
```

it does not work as expected.

Reply:
See also https://github.com/nodejs/node-v0.x-archive/issues/9271